services/wfm/src/main/java/org/openkilda/wfm/topology/islstats/IslStatsTopology.java

```diff
@@ -40,15 +40,15 @@ public StormTopology createTopology() {
         String topoDiscoTopic = topologyConfig.getKafkaTopoDiscoTopic();
 
         logger.debug("connecting to {} topic", topoDiscoTopic);
-        builder.setSpout(ISL_STATS_SPOUT_ID, createKafkaSpout(topoDiscoTopic, ISL_STATS_SPOUT_ID));
+        builder.setSpout(ISL_STATS_SPOUT_ID, buildKafkaSpout(topoDiscoTopic, ISL_STATS_SPOUT_ID));
 
-        IslStatsBolt verifyIslStatsBolt = new IslStatsBolt();
+        IslStatsBolt verifyIslStatsBolt = new IslStatsBolt(topologyConfig.getMetricPrefix());
         logger.debug("starting {} bolt", ISL_STATS_BOLT_ID);
         builder.setBolt(ISL_STATS_BOLT_ID, verifyIslStatsBolt, topologyConfig.getParallelism())
                 .shuffleGrouping(ISL_STATS_SPOUT_ID);
 
         String openTsdbTopic = topologyConfig.getKafkaOtsdbTopic();
-        KafkaBolt openTsdbBolt = createKafkaBolt(openTsdbTopic);
```

Comment:
please do not change `createKafkaBolt` and `createKafkaSpout` yet. It causes `Could not deserialize message=00:00:00:00:00:00:00:07`

we need to fix #1985 first

Reply:
Fixed.